diagnostic.md

```diff
@@ -56,8 +74,8 @@ let answer = 'not empty'
 
 while (answer !== '') {
   answer = ask("Guess my secret? ")
-  if (answer === 'SeCrEt') {
-   console.log("You guessed it!")
+  if (answer === < 0 && > 11) {
+   console.log("Guess needs to be between 1 and 10.")
```

Comment:
Looks good, but make sure that you actually are giving users a number to guess in addition to making sure the numbers are between 1-10